upload_to_s3.go

```diff
@@ -0,0 +1,80 @@
+package main
+
+import (
+	"context"
+	"log"
+	"os"
+	"path/filepath"
+
```

Comment:
Is this space intentional for diving current folders from URLs or can be removed?

Reply:
Yes, its the space between builtin packages and 3rd party packages